compiler/src/iree/compiler/Codegen/Dialect/IREECodegenAttrs.cpp

```diff
@@ -320,12 +338,62 @@ LogicalResult setDispatchConfig(func::FuncOp entryPoint,
   return success();
 }
 
+static void createIncludeTransformStrategy(func::FuncOp entryPoint,
```

Comment:
Not sure what this is for.

Reply:
Basically we need to create a transform region for the interpreter to run that ties the `hal.executable.variant` op handle passed as input to the strategy specified by the symbol ref. This might not be the best place for this function but we need to create the `transform.sequence` op somewhere. Part of the reason we actually want to materialize the sequence instead of changing upstream to allow passing in a named symbol directly is that this is the stage at which we will want to initialize the static parameters that could be passed in as inputs to the strategy (e.g. tile sizes).